lib/l10n/app_en.arb

```diff
@@ -525,8 +526,10 @@
   "create_video_edit_cover": "Edit cover",
   "create_video_new_video": "New video",
   "create_video_input_placeholder": "Add a description (optional)",
-  "gallery_add_photo_title": "Add photo",
-  "gallery_add_media_title": "Add media",
+  "gallery_all_photos_title": "All photo",
+  "gallery_all_media_title": "All media",
+  "gallery_albums_all_title": "All",
```

Comment:
`core_all`
Could we use it to avoid creating duplicate translations?

Reply:
done